### PR TITLE
[refactor](scan) optimize the agg function of count(1)

### PR DESCRIPTION
**Pull request by zhangstar333**
# Proposed changes
after PR #12803,
like count(*) in dup table will use the segment static,
now add count(const value) also use that optimization.

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/planner/SingleNodePlanner.java [hunk 1]
```diff
@@ -490,8 +493,7 @@ private void pushDownAggNoGrouping(AggregateInfo aggInfo, SelectStmt selectStmt,
                                 break;
                             }
 
-                            if (colType.isCharFamily() && aggOp != TPushAggOp.COUNT
-                                    && col.getType().getLength() > 512) {
+                            if (colType.isCharFamily() && col.getType().getLength() > 512) {
```
Comment (HappenLee) [inline fe/fe-core/src/main/java/org/apache/doris/planner/SingleNodePlanner.java:496]: why change this ？

**Review by HappenLee — DISMISSED** [review]
LGTM

**Comment by zhangstar333** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 33.05 seconds
 stream load tsv:          430 seconds loaded 74807831229 Bytes, about 165 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          60 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230417095803_clickbench_pr_130089.html

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run p1

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**